### PR TITLE
fix minor issues

### PR DESCRIPTION
**Pull request by filiatra**

### CODE REVIEW

**Comment by filiatra** [comment]
It is bad practice to call exit(), and we need to elimintate it. Instead, for more beautiful message one can handle the exception as in the commit for gsView..

File: external/tclap/CmdLine.h [hunk 1]
```diff
@@ -504,7 +504,8 @@ inline void CmdLine::parse(std::vector<std::string>& args)
 	}
 
 	if (shouldExit)
-		exit(estat);
+        throw(CmdLineParseException("Command line argument exception", gismo::util::to_string(estat))); // G+Smo
```
Comment (stefantakacs) [inline external/tclap/CmdLine.h]: It is supposed to be a clean exit; we might consider (re-)throwing ExitException.

**Comment by stefantakacs** [comment]
// add to gsCmdLine.h

class gsExitException {
    int exit_code;
public:
    gsExitException(int _exit_code) : exit_code(_exit_code) {}
    operator int ()    { return exit_code;         }
    const char* what() { return "gsExitException"; }
};

// callers invoke:

try { cmd.getValues(argc,argv); } catch( gsExitException e ) { return e; }

Then we could even preserve the current behavior that --help and --version yield
exit code 0.

**Comment by stefantakacs** [comment]
@filiatra it is up to you if you want to merge this or a previous version

**Comment by stefantakacs** [comment]
any thoughts about this pr?

**Comment by filiatra** [comment]
I changed it to just through the exit status, look fine to me now

**Comment by stefantakacs** [comment]
Yes, this is also a good solution.

I will update devel after you have merged this.

**Pull request merged**